explorer/client/src/components/KeyValueList.tsx

```diff
@@ -0,0 +1,108 @@
+import Card from '@material-ui/core/Card'
```

Comment:
Had to manually copy this component over from the storybook. Can remove once...
https://www.pivotaltracker.com/story/show/167792209
...is completed

Reply:
Is this fixed by https://github.com/smartcontractkit/chainlink/pull/2067 ?

Reply:
I believe #2067 tests are failing because of the same underlying issue

Reply:
@se3000 - I think it's [this story](https://www.pivotaltracker.com/story/show/167792209) that is blocking the use of the storybook. Can you confirm @HenryNguyen5?